core/domain/takeout_service.py

```diff
@@ -18,15 +18,43 @@
 from __future__ import absolute_import   # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
+import inspect
 import re
 
 from core.platform import models
 
 (
-    collection_models, email_models, exploration_models, feedback_models,
-    suggestion_models, user_models) = models.Registry.import_models([
-        models.NAMES.collection, models.NAMES.email, models.NAMES.exploration,
-        models.NAMES.feedback, models.NAMES.suggestion, models.NAMES.user])
+    base_models, collection_models, email_models, exploration_models,
+    feedback_models, suggestion_models,
+    user_models) = models.Registry.import_models(
+        [models.NAMES.base_model, models.NAMES.collection, models.NAMES.email,
+         models.NAMES.exploration, models.NAMES.feedback,
+         models.NAMES.suggestion, models.NAMES.user])
+
+
+def get_models_to_export():
+    """Returns set of all models whose data to export."""
```

Comment:
Missing Returns docstring.

The return value doesn't look like a set of models, it looks like a set of strings. Could you please clarify?

Reply:
Done